### DISCUSSION TITLE
Wrong(Big) Number of jobs assigned to worker by miner

### DISCUSSION THREAD

**Discussion by hcm609** [filecoin-project/lotus#7731] [Storage Provider] [answered]
Hi.
I have the worker (c18) that was the only one connected to miner.
The configuration in c18 worker has the following lines:
[Storage]
  ParallelFetchLimit = 6
[Sealing]
  MaxWaitDealsSectors = 6
  MaxSealingSectors = 6
  MaxSealingSectorsForDeals = 6

Despite this, 50 AP jobs were assigned by the miner to the worker overloading it without favouring the limit of 6 simultaneous jobs/fetches.

ID        Sector  Worker    Hostname  Task  State        Time
27124e02  1409    00000000  c14       PC1   ret-wait     1h8m10.2s
a8e8ad32  2093    00000000  c4        C2    ret-wait     55m37.2s
830df734  1072    07c0e7ca  c18       AP    running      17m14s
fdbbdc95  1063    07c0e7ca  c18       AP    running      17m14s
276f86e7  1060    07c0e7ca  c18       AP    running      17m14s
92ea301a  1061    07c0e7ca  c18       AP    running      17m13.9s
7d3be6ea  1062    07c0e7ca  c18       AP    running      17m13.9s
f0490644  1069    07c0e7ca  c18       AP    running      17m13.8s
4c3d970b  1067    07c0e7ca  c18       AP    running      17m13.8s
d5cb77af  1068    07c0e7ca  c18       AP    running      17m13.7s
8e74de66  1065    07c0e7ca  c18       AP    running      17m13.7s
13c2f867  1066    07c0e7ca  c18       AP    running      17m13.6s
d09e3b07  1064    07c0e7ca  c18       AP    running      17m13.5s
e20326e2  1070    07c0e7ca  c18       AP    running      17m13.4s
70871ce8  1071    07c0e7ca  c18       AP    running      17m13.2s
b8e98582  1073    07c0e7ca  c18       AP    running      16m51.7s
02dd2944  1077    07c0e7ca  c18       AP    running      16m51.6s
c32716a4  1076    07c0e7ca  c18       AP    running      16m51.5s
6a4ff29f  1083    07c0e7ca  c18       AP    running      16m51.4s
a141703f  1079    07c0e7ca  c18       AP    running      16m51.2s
093baa00  1080    07c0e7ca  c18       AP    running      16m51s
43b4e30e  1075    07c0e7ca  c18       AP    running      16m50.9s
75ecff17  1084    07c0e7ca  c18       AP    running      16m50.9s
b6827b7a  1081    07c0e7ca  c18       AP    running      16m50.8s
48cbdb2d  1078    07c0e7ca  c18       AP    running      16m50.7s
a801306a  1085    07c0e7ca  c18       AP    running      16m50.6s
124f1fdb  1082    07c0e7ca  c18       AP    running      16m50.5s
94dba2be  1086    07c0e7ca  c18       AP    running      16m50.2s
d54e24a4  1101    07c0e7ca  c18       AP    running      16m49.6s
3276a116  1091    07c0e7ca  c18       AP    running      16m49.6s
2f69bab9  1092    07c0e7ca  c18       AP    running      16m49.6s
f5443c22  1102    07c0e7ca  c18       AP    running      16m49.5s
914bd03f  1094    07c0e7ca  c18       AP    running      16m49.5s
52b85d58  1096    07c0e7ca  c18       AP    running      16m49.4s
15d6e165  1090    07c0e7ca  c18       AP    running      16m49.4s
720d79b6  1093    07c0e7ca  c18       AP    running      16m49.3s
922fdf52  1099    07c0e7ca  c18       AP    running      16m49.2s
2a0ec259  1088    07c0e7ca  c18       AP    running      16m49.2s
246290fc  1098    07c0e7ca  c18       AP    running      16m49.1s
de343925  1087    07c0e7ca  c18       AP    running      16m49.1s
e1010aaf  1105    07c0e7ca  c18       AP    running      16m49s
738c21e4  1107    07c0e7ca  c18       AP    running      16m49s
49e70e32  1089    07c0e7ca  c18       AP    running      16m48.9s
2537e2f2  1104    07c0e7ca  c18       AP    running      16m48.9s
1f7fd2a6  1106    07c0e7ca  c18       AP    running      16m48.9s
433cae1d  1103    07c0e7ca  c18       AP    running      16m48.8s
d3837635  1108    07c0e7ca  c18       AP    running      16m48.8s
2dee1b4d  1100    07c0e7ca  c18       AP    running      16m48.7s
290e188f  1110    07c0e7ca  c18       AP    running      16m48.7s
9407a7c5  1097    07c0e7ca  c18       AP    running      16m48.7s
7f8b89a4  1095    07c0e7ca  c18       AP    running      16m48.6s
d49e8c79  1109    07c0e7ca  c18       AP    running      16m48.6s
9bf95bc5  1111    07c0e7ca  c18       AP    running      16m48.6s
2b2111af  1120    07c0e7ca  c18       AP    running      16m48.5s

**Comment by SgtCoin** [comment]
Has anything changed since this happened? If you restart the worker again, does it happen a second time?

**Reply by hcm609** [reply]
no change. I see the same behaviour like the scheduler of the miner is problematic.

**Comment by SgtCoin** [comment]
What version of lotus are you running? The latest version includes enhancements to the scheduling processes.

**Comment by SgtCoin** [comment, marked as answer]
Archiving due to no response.
